TestAssets/NonRestoredTestProjects/PJAppWithSlnAndOneAlreadyMigratedCsproj/TestLibrary/TestLibrary.csproj

```diff
@@ -4,6 +4,7 @@
     <TargetFramework>netstandard1.5</TargetFramework>
     <AssemblyName>TestLibrary</AssemblyName>
     <PackageId>TestLibrary</PackageId>
+    <RestoreSources>https://api.nuget.org/v3/index.json</RestoreSources>
```

Comment:
Is this because we wouldn't ever expect to use alternative restore sources for these test projects?

Reply:
Should this be conditionalized this on offline build too?

Reply:
Would an offline 'build' run tests?